content/04.results.md

```diff
@@ -9,5 +9,36 @@ place the grid of aurocs here for discriminator model
 ## Random Noise Generative Model
 place the results of random label function experiment
 
-## Reconstructing Hetionet
-place figure of number of new edges that can be added to hetionet as well as edges we can reconstruct using this method
+
+### Reconstructing Hetionet
+Once the discriminiative model has been calibrated, we compared our model to a baseline model [@tag:cocoscore] (Figure {@fig:cocoscore_comparison}) and  estimated the number of edges that can be added/recalled from Hetionet v1 (Figure Figure {@fig:hetionet_reconstruction}).
+Regarding model comparison we evaluated edge prediction performance using all edges assigned to our test set.
+Scores for both models are reported in the form of area under the receiver operative curve (AUROC) and area under the precision recall curve (AUPR).
+The baseline model outperforms our model for majority of edge types.
+Our model achieves an AUROC of 78% and an AUPR of 28% for Disease-associates-Gene (DaG), while the baseline model achieves a score of 82% and 45% resepctively.
+For Gene-interacts-Gene the baseline model achieved an AUROC of 73% and an AUPR of 11%, while our model achieved an AUROC of 72% and an AUPR of 7%.
+Our model out performed the literature model for Compound-treats-Disease (CtD) by obtaining an AUROC of 94% and an AUPR of 40%.
```

Comment:
All of these things are apparent from the figure. You don't need to repeat things that are exactly in the figure in a text form. Can you summarize the main interpretation here? See what I did in #15 for discriminative model performance.